packages/theme/src/colors.ts

```diff
@@ -92,6 +92,8 @@ const light = {
     borderElevationNegative: palette.lightGray400,
     borderSecondary: palette.lightSecondaryEmerald800,
     borderSubtleInverted: palette.lightWhiteAlpha400,
+    borderInputDefault: palette.lightGray400,
```

Comment:
Where does these two new colors come from? I can not find them anywhere in the design files.  Did @shenkys want to add these colors?

Otherwise looks good 👌.

Reply:
Yep, @shenkys asked for this change. I guess he forgot to update the ticket as well.